packages/snaps-jest/src/internals/simulation/interface.test.tsx

```diff
@@ -191,41 +197,50 @@ describe('getInterfaceResponse', () => {
 
     expect(() => {
       // @ts-expect-error - Invalid dialog type.
-      getInterfaceResponse(runSaga, 'Foo', text('foo'));
+      getInterfaceResponse(runSaga, 'Foo', <Text>foo</Text>);
     }).toThrow('Unknown or unsupported dialog type: "Foo".');
   });
 });
 
 describe('getElement', () => {
   it('gets an element at the root', () => {
     const content = button({ value: 'foo', name: 'bar' });
-
-    const result = getElement(content, 'bar');
+    const result = getElement(getJsxElementFromComponent(content), 'bar');
 
     expect(result).toStrictEqual({
-      element: button({ value: 'foo', name: 'bar' }),
+      element: <Button name="bar">foo</Button>,
     });
   });
 
   it('gets an element with a given name inside a panel', () => {
     const content = panel([button({ value: 'foo', name: 'bar' })]);
-
-    const result = getElement(content, 'bar');
+    const result = getElement(getJsxElementFromComponent(content), 'bar');
 
     expect(result).toStrictEqual({
-      element: button({ value: 'foo', name: 'bar' }),
-      form: undefined,
+      element: <Button name="bar">foo</Button>,
     });
   });
 
   it('gets an element in a form', () => {
     const content = form('foo', [button({ value: 'foo', name: 'bar' })]);
+    const result = getElement(getJsxElementFromComponent(content), 'bar');
 
-    const result = getElement(content, 'bar');
+    expect(result).toStrictEqual({
+      element: <Button name="bar">foo</Button>,
+      form: getJsxElementFromComponent(content),
+    });
+  });
+
+  it('gets an element in a form when there are multiple forms', () => {
+    const content = panel([
+      form('foo', [button({ value: 'foo', name: 'bar' })]),
+      form('foo', [button({ value: 'foo', name: 'baz' })]),
+    ]);
+    const result = getElement(getJsxElementFromComponent(content), 'baz');
 
     expect(result).toStrictEqual({
-      element: button({ value: 'foo', name: 'bar' }),
-      form: 'foo',
+      element: <Button name="baz">foo</Button>,
+      form: getJsxElementFromComponent(content.children[1]),
```

Comment:
Shouldn't this be returning a string for `form`? Also two forms with the same name is invalid

Reply:
I refactored the entire function and changed it to return a form element, but I suppose it's unnecessary.